components/src/core/app-bar/app-bar.component.scss

```diff
@@ -0,0 +1,12 @@
+.pxb-app-bar-content {
+    top: 0;
+    position: static;
+    min-height: unset;
+    display: flex;
+    justify-content: center;
+    align-items: center;
```

Comment:
I think we should keep these styles so that the content is center aligned in the expanded toolbar.

Whenever the toolbar is collapsed we don't need these styles.    Whenever I add the 3-liner component I might remove these styles if we can.

Reply:
Since our expanding toolbar can (should) technically be able to support any content the user wants, I would say it would be better to force as few styles as possible on the container and let them control the positioning in their content. But if you don't want to do it now, I'll comment again later :)

E.g. the three liner component should have `height: 100%` and then set up its own flexbox properties to put the content where it needs to be.